### ISSUE TITLE
Can't correctly pad a character in a smiles and regenerate padded smiles using cddd

### ISSUE THREAD

**Issue by swapchavan** [jrwnter/cddd#22]
I was trying to pad N6 in aniline

```
import os
import sys
from IPython.display import SVG
import torch
from rdkit import Chem
from rdkit.Chem import AllChem
from rdkit.Chem import Draw
from rdkit.Chem.Draw import rdMolDraw2D

sys.path.insert(0, './cddd')
from cddd.inference import InferenceModel 
inference_model = InferenceModel()

aniline_smi = "C1=CC=C(C=C1)N"
m = Chem.MolFromSmiles(aniline_smi)
edit_m = Chem.RWMol(m)
# Replace N6 by Yttrium and then replace Yttrium by padding char
edit_m.GetAtomWithIdx(6).SetAtomicNum(39)
masked_N6_aniline_smiles = Chem.MolToSmiles(edit_m).replace('[Y]','</s>')
print(masked_N6_aniline_smiles)
'</s>c1ccccc1'
# convert smiles to embedding
cddd_descr_masked_aniline = inference_model.seq_to_emb(masked_N6_aniline_smiles)
print(cddd_descr_masked_aniline.shape)
# regenerate  smiles from embedding
regenerate_masked_aniline = inference_model.emb_to_seq(torch.tensor(cddd_descr_masked_aniline).double())
print(regenerate_masked_aniline)
'Sc1ccccc1'

```

Why cddd doesn't return the padded-smiles? Why does it convert padding character to sulphur?
I tried to pad middle characters too, but it always generated wrong smiles. Can I mask any character in my smiles and generate cddd descriptors, and then convert it back to the original padded-smiles?
Your kind answer will be deeply regarded!

**Comment by jrwnter** [comment]
By default, the preprocessing ignores characters it doesn't know ( e.g. "<", "/", ">"). Hence, "</s>" is parsed as "s" which is the token for sulfur. However, if you just want to ignore Yttrium, you can either remove it from the string or keep it, as it will be ignored anyway...
